src/occ_map/voxel_map.hpp

```diff
@@ -0,0 +1,118 @@
+// Copyright 2017 Massachusetts Institute of Technology
+#pragma once
+
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <math.h>
+#include <assert.h>
+
+#define NO_LCM
+
+namespace occ_map {
+
+template<class T>
+class VoxelMap {
+public:
+  //metadata
+  double xyz0[3], xyz1[3];
+  double metersPerPixel[3];
+  int dimensions[3];
+  int num_cells;
+
+#ifndef NO_LCM
+  occ_map_voxel_map_t *msg;
+#endif
+  int64_t utime;
+
+  //the actual storage arrays
+  T * data;
```

Comment:
Am going to pass on messing with the underlying occ_map library in this PR.  A big refactor/rewrite is in its future, though.